instance/templates/instance/ansible/vars.yml

```diff
@@ -75,3 +75,5 @@ COMMON_USER_INFO:{% for github_username in github_admin_username_list %}
   - name: {{ github_username }}
     github: true
     type: admin{% endfor %}{% endif %}
+
+edxapp_worker_default_stopwaitsecs: 60
```

Comment:
@e-kolpakov Nit: It would be nice to put this in a dedicated section (i.e., add a comment on top, such as `# Workers`).